NetKAN/ImageViewerCont.netkan

```diff
@@ -0,0 +1,14 @@
+{
+  "spec_version": "v1.4",
+  "install": [
+    {
+      "install_to": "GameData",
+      "find": "ImageViewer"
+    }
+  ],
+  "$vref": "#/ckan/ksp-avc",
```

Comment:
If there are multiple version files in a download, ckan doesn't know which one to pick and fails. Adding a path to the vref tells which file to use.